compiler-plugin/src/main/scala/com/trueaccord/scalapb/compiler/ProtobufGenerator.scala

```diff
@@ -382,8 +382,14 @@ class ProtobufGenerator(val params: GeneratorParams) extends DescriptorPimps {
       .add("}")
       .add("final override def serializedSize: Int = {")
       .indent
-      .add("if (__serializedSizeCachedValue == -1) { __serializedSizeCachedValue = __computeSerializedValue() }")
-      .add("__serializedSizeCachedValue")
+      .add("var read = __serializedSizeCachedValue")
```

Comment:
It's two field accesses vs one.

JDK's toString hashCode and protobuf's ByteString use similar style:
https://github.com/google/protobuf/blob/master/java/core/src/main/java/com/google/protobuf/ByteString.java#L842

But I can make it back if you still want.